multisig/proposals/opsepolia/vip-007/index.ts

```diff
@@ -0,0 +1,71 @@
+import { parseUnits } from "ethers/lib/utils";
+import { NETWORK_ADDRESSES, ZERO_ADDRESS } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { opsepolia } = NETWORK_ADDRESSES;
+
+export const ACM = "0x1652E12C8ABE2f0D84466F0fc1fA4286491B3BC1";
+export const REWARD_DISTRIBUTOR_CORE_0 = "0x24139Dad3fe87Ee718ff9c2A8E0C4188578ba9aF";
+export const COMPTROLLER_CORE = "0x59d10988974223B042767aaBFb6D926863069535";
+export const XVS_VAULT_REWARDS_SPEED = "31709791983764"; // (1,000 XVS per year)
+export const XVS_STORE = "0xE888FA54b32BfaD3cE0e3C7D566EFe809a6A0143";
+
+export const VWBTC_CORE = "0x6149eFAd7671f496C900B3BeC16Ba31Aed60BE4b";
+export const VWETH_CORE = "0x4E610626BeF901EEE22D558b2ed19e6f7B87cf51";
+export const VUSDC_CORE = "0x2419606690B08060ebFd7581e0a6Ae45f1915ee9";
+export const VUSDT_CORE = "0xC23D18536E7069f924B3717B2710CA6A09e53ea9";
+export const VOP_CORE = "0x49cceCdd0b399C1b13260452893A3A835bDad5DC";
+export const XVS_STORE_AMOUNT = parseUnits("5000", 18);
+
+const vip007 = () => {
+  return makeProposal([
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLastRewardingBlocks(address[],uint32[],uint32[])", opsepolia.GUARDIAN],
+    },
```

Comment:
I don't think we need this permission, because it's a time-based contract

Reply:
fixed, https://github.com/VenusProtocol/vips/pull/385/commits/91ae6235d7c1d842e103406c933ed9f1eaca8163